src/operator/tensor/elemwise_unary_op.cc

```diff
@@ -344,9 +344,9 @@ MXNET_OPERATOR_REGISTER_BINARY(_backward_expm1)
 
 // cos
 MXNET_OPERATOR_REGISTER_UNARY(cos)
-.describe(R"code(Cosine, element-wise.
+.describe(R"code(Returns element-wise trigonometric cosine value of the input.
```

Comment:
Change to "Computes the element-wise cosine of the input array."

Note: "trigonometric cosine" is redundant. Does anyone know of a different cosine? Even 12 year olds have to learn cosine, so this one I think is safe to assume.